src/routes/users.route.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+
+const { authMiddleware } = require('../middlewares/authMiddleware');
+
+const { usersControllers } = require('../controllers/users.controller');
```

Comment:
Ensure that `usersControllers` is correctly imported from the `users.controller` file and contains the `getUsers` and `updateUser` methods.